pkg/controller/scyllacluster/resource.go

```diff
@@ -31,11 +31,14 @@ const (
 )
 
 func HeadlessServiceForCluster(c *scyllav1.ScyllaCluster) *corev1.Service {
+	labels := naming.ClusterLabels(c)
+	labels[naming.ScyllaServiceTypeLabel] = string(naming.ScyllaServiceTypeIdentity)
```

Comment:
would you mind splitting service type label changes to separate PR? It's worth notiifying users about the change in the release notes.

Reply:
#969